app/styles/page-not-found.css

```diff
@@ -0,0 +1,13 @@
+.page-not-found{
+    margin: auto;
+    text-align: center;
+  }
+  
+  .page-not-found__img {
+    width: 20%;
+  }
+  
+  .page-not-found__text{
+    font-size: 15px;

```

Comment:
We can remove the font-size and font-weight properties from here and instead use the `h4 tag` whose font size is 16px and font-weight: bolder.